components/usage/pkg/contentservice/client.go

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package contentservice
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/gitpod-io/gitpod/content-service/api"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+type Interface interface {
```

Comment:
Do we need this wrapper? Wouldn't using the `api.UsageReportServiceClient` directly not be sufficient?

Reply:
In practice, you set this up once during server start and then inject the client into the controller which means this abstraction doesn't add much.

Reply:
How would you then handle the no-op case where we don't want to upload to cloud storage when no `contentServiceUrl` is configured?

The reason for this `Interface` was just to be able to do that - have a real implementation and a no-op one.

Reply:
Ah, good point. Makes perfect sense to have it then.